main/__tests__/ButtonGroup.test.tsx

```diff
@@ -10,11 +10,11 @@ describe('[ButtonGroup]', () => {
 
   const handlePress = jest.fn();
 
-  const renderButtonGroup = (): RenderAPI =>
+  const renderButtonGroup = (index?: number): RenderAPI =>
     render(
       <ThemeProvider initialThemeType={themeType}>
         <ButtonGroup
-          initialIndex={0}
+          selected={index}
```

Comment:
Looks like the props here is not changed yet `selectedIndex`

Reply:
Sorry for that. I made commit to fix this.

Reply:
This happens due to insufficient test that didn't caught my mistake.
I update tests to test initial value of selectedIndex. and fix mistake.